cosmic-core/test/integration/smoke/test_ip_exclusion_list.py

```diff
@@ -25,96 +25,13 @@
 
 class TestIpExclusionList(cloudstackTestCase):
 
-    attributes = {
-        'account': {
-            'email': 'e.cartman@southpark.com',
-            'firstname': 'Eric',
-            'lastname': 'Cartman',
-            'username': 'e.cartman',
-            'password': 'southpark'
-        },
-        'vpcs': {
-            'vpc1': {
-                'name': 'vpc1',
-                'displaytext': 'vpc1',
-                'cidr': '10.1.0.0/16'
-            }
-        },
-        'networks': {
-            'network1': {
-                'name': 'network1',
-                'displaytext': 'network1',
-                'gateway': '10.1.1.1',
-                'netmask': '255.255.255.0'
-            },
-            'network2': {
-                'name': 'network2',
-                'displaytext': 'network2',
-                'gateway': '10.1.2.1',
-                'netmask': '255.255.255.248'
-            }
-        },
-        'vms': {
-            'vm1': {
-                'name': 'vm1',
-                'displayname': 'vm1'
-            },
-            'vm2': {
-                'name': 'vm2',
-                'displayname': 'vm2',
-                'privateport': 22,
-                'publicport': 22,
-                'protocol': 'TCP'
-            },
-            'vm3': {
-                'name': 'vm3',
-                'displayname': 'vm3',
-                'privateport': 22,
-                'publicport': 22,
-                'protocol': 'TCP'
-            }
-        },
-        'nat_rule': {
-            'protocol': 'TCP',
-            'publicport': 22,
-            'privateport': 22
-        },
-        'acls': {
-            'acl1': {
-                'name': 'acl1',
-                'description': 'acl1',
-                'entries': {
-                    'entry1': {
-                        'protocol': 'TCP',
-                        'action': 'Allow',
-                        'traffictype': 'Ingress',
-                        'startport': 22,
-                        'endport': 22
-                    }
-                }
-            },
-            'acl2': {
-                'name': 'acl2',
-                'description': 'acl2',
-                'entries': {
-                    'entry2': {
-                        'protocol': 'TCP',
-                        'action': 'Deny',
-                        'traffictype': 'Ingress',
-                        'startport': 22,
-                        'endport': 22
-                    }
-                }
-            }
-        }
-    }
-
     @classmethod
     def setUpClass(cls):
         cls.logger = MarvinLog(MarvinLog.LOGGER_TEST).get_logger()
 
         cls.test_client = super(TestIpExclusionList, cls).getClsTestClient()
         cls.api_client = cls.test_client.getApiClient()
+        cls.test_data = cls.test_client.getParsedTestDataConfig().copy()
```

Comment:
@iri989 Added a commit to comply with your requested changes. IMHO your review can be dismissed.